examples/Makefile

```diff
@@ -20,9 +20,9 @@ release debug:
 	$(MAKE) embedding-$@
 
 %.o: %.c
-	@$(call PRINT_CC, $(CC) $(CPPFLAGS) $(CFLAGS) $(SHIPFLAGS) -c $< -o $@)
+	@$(call PRINT_CC, $(CC) $(CPPFLAGS) $(JCPPFLAGS) $(CFLAGS) $(JCFLAGS) $(SHIPFLAGS) -c $< -o $@)
```

Comment:
I believe the intention was to have this file be relatively standalone, which is why these weren't put here. Maybe we should use `./julia ./contrib/julia-config.jl --cflags` here instead, so that we are also dog-fooding that script?